ci/sdk_go.go

```diff
@@ -138,6 +139,7 @@ func gitPublish(ctx context.Context, opts gitPublishOpts) error {
 		base = dag.Container().From(consts.AlpineImage)
 	}
 
+	// FIXME: move this into std modules
```

Comment:
There's a few `FIXME`s in here - is the aim to do these before/after PR merge?

Reply:
After. There is a lot to do in this code base, and honestly it's quite painful to gain momentum. I would really prefer getting a first change in, and keep iterating from there. I'm already preparing my next patch.